### PR TITLE
Added test and formatting

### PR DESCRIPTION
**Pull request by julius-rennmann**
I have added the jest tests as discussed.

All tests passed. I do not know how useful the tests for an exact match of the results to a did document are, but I left them there for now.

I also ran prettier on all of the code. These changes are only to formatting and do not change any behavior.
I also patched the version to 0.1.3. If everything is alright i would then publish this version to npm.

### CODE REVIEW

File: test/resolver.test.ts [hunk 1]
```diff
@@ -1,53 +1,55 @@
...
-      console.log('Error resolving DID:', e);
-    }
-  })
-
+  const responses = testDids.map(({ did, expectedResult }) => ({
+    did,
+    expectedResult,
+    asyncResponse: resolver.resolve(did, { fetch }),
+  }));
+  describe('Dids resolve to DIDResolutionResult', () => {
```
Comment (theblockstalk) [inline test/resolver.test.ts:15]: please use async await notation, not promise.then

**Review by theblockstalk — COMMENTED** [review]
looks good in principal. please use `async` `await` instead of promises().then()...

**Comment by julius-rennmann** [comment]
Ok all done

File: test/resolver.test.ts (hunk 1 above)
Comment (theblockstalk) [inline test/resolver.test.ts:15]: does this resolve each DID test several times (one time for each `describe`) or only once and the result is then inspected several times (in each `describe`)?

It should be the latter.

Comment (julius-rennmann) [reply test/resolver.test.ts:15]: Every did is resolved only once, the result is then tested multiple times. There are 4 tests for 7 dids, so 28 tests for the resolver. The resolver is called 7 times.

**Pull request merged**